src/components/Loading.tsx

```diff
@@ -1,7 +1,22 @@
-export default function Loading() {
+type LoadingProps = {
+  isButton?: boolean;
```

Comment:
I dont think this is actually used anywhere else, the `Loading` component, so we could probably just make this only apply to buttons

Reply:
Sounds good.  I wasn't sure if this was planned to be use elsewhere.  Removed the isButton logic/prop